projects/igniteui-angular/src/lib/core/styles/components/chip/_chip-theme.scss

```diff
@@ -432,11 +432,11 @@
         align-items: center;
         z-index: 1;
         opacity: 1;
-        transition: opacity 120ms $ease-in-out-quad;
+        width: 18px;
```

Comment:
This approach breaks the chip in Safari. I don't have the means to currently test it in IE and old Edge.

Reply:
Please, suggest approach sir :)